client/src/modules/vouchers/toolkit/support_patient.modal.html

```diff
@@ -0,0 +1,60 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>{{ ToolCtrl.tool.label }}</li>
+  </ol>
+</div>
+
+<form name="ToolForm" bh-submit="ToolCtrl.import(ToolForm)" bh-form-defaults novalidate>
```

Comment:
`bhFormDefaults` doesn't work, unfortunately.  The Chrome browser team does not allow blocking autocomplete.

See https://github.com/IMA-WorldHealth/bhima-2.X/issues/1447 for more details.